cli/dcos-spark/main.go

```diff
@@ -23,22 +23,25 @@ type SparkCommand struct {
 	submitArgs        string
 	submitDockerImage string
 	submitEnv         map[string]string
-	secretPath		  string
-
+	secretPath 		  string
```

Comment:
I just ran `gofmt -w` on this file locally, and this fixes the formatting issues in this block and doesn't change anything else. I would recommend doing that.

Reply:
Ah yes! `gofmt` ... I'm still a golang newb so I forget about this but I do love it. I used to get into arguments with other devs over formatting whitespace in Scala/Java and here go embraces it. A++++